### ISSUE TITLE
OBS Stream monitor

### ISSUE THREAD

**Issue by koskit** [louislam/uptime-kuma#2271]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Monitor

### 🔖 Feature description

It would be nice to have a feature for checking stream status/quality for uptime and network congestion (these metrics already exists within obs). All there is to do, is retrieve them and if a specific threshold is passed (more that 60 missed frames etc), send the notification and be able to see the stream-health history on uptime kuma.

### ✔️ Solution

For example, I have some streams (with OBS) that might be laggy at some point, and/or go down because of network issues.

What I had in mind was a Specific monitor type that integrates with OBS-WebSockets (now that the WebSocket feature is embedded in the latest OBS version and no need for plugins) that will ask the OBS stream what is it's status or be a passive monitor type that OBS sends to uptime kuma the statuses (with events?). 

### ❓ Alternatives

I wrote a C# service that checks all of the above and sends me a notification if things are looking bad, but I was missing the histogram of the stats. I had no way (without huge effort) to check what was the status some minutes prior etc. 

When searching for some self-hosted uptime tracker came across uptime kuma. 

What could be done (to make it work without all the hassle of a new feature, is to create a passive monitor type that sends whether the stream is running correctly, but I found no parameters to pass the extra stuff (like the _Optional parameters: status, msg, ping_ that exist in the Monitor Type -> Passive Monitor Type -> Push).

I could pass the info I wanted as msg and/or ping, but I wouldn't be able to create rules on them. E.g. if msg < 3000 (lets say I send the bitrate to msg), send me a notification that the stream has some network congestion.

### 📝 Additional Context

Something like the following:

![image](https://user-images.githubusercontent.com/4030434/197820568-7b161402-e2b8-434e-9d39-416c37c8107c.png)

PS. I get that uptime kuma is more oriented for web applications, but I would consider a stream as a service in my home that I would like to "monitor". 

Basically we still check the uptime of something, but instead of latency we have missed frames and/or network congestion (if there was something else "eating" the bandwidth), and we should send a notification in case the are connection issues.
